### PR TITLE
added hmr-it-controlling-bundle

### PR DESCRIPTION
**Pull request by hmr-it-jr**
Added a new bundle which I've developed.

A plugin for Kimai 2 which checks if one of your users violated the German break time law + nominal and actual work time comparison + overtime view.

### CODE REVIEW

**Comment by hmr-it-jr** [comment]
@kevinpapst now I'm done with adding commits 😄 

If I need to change something please write me

**Comment by kevinpapst** [comment]
awesome 👍 will have a look later

File: _store/hmr-it-controlling-bundle.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+- See if your users have overtimes or undertimes (since first day in company, per month, per day - all in separate views)
+- Check if they violate the German break time law "Arbeitszeitgesetz (ArbZG) § 4 Ruhepausen"
+- Users see how much overtimes or undertimes they have + how much they've worked already in the current month and how much they still have to work in the current month
+- Filter data per user, time period, and so on
+- E-Mail to user to notify him, that he has violated the German break time law
+- Overview for Administrators to see all calculated times in one view
+- Get notified if one of your user's birthday is coming soon
+- See all information in the dashboard
+- Get notified if one of your user has a anniversaries like 5 years in company
+- Handout for employees as pdf to see the worked hours of the month
```
Comment (kevinpapst) [inline _store/hmr-it-controlling-bundle.md:65]: pdf = PDF

File: _store/hmr-it-controlling-bundle.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+- Handout downloadable as a single pdf file for all employees or easy sendable via email to each employee (even to his private email account!)
```
Comment (kevinpapst) [inline _store/hmr-it-controlling-bundle.md:66]: pdf = PDF

File: _store/hmr-it-controlling-bundle.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+- Detect overlapping times with one click
+- See if your users have missing days (no times entered on days where they had to work)
+- Fully configurable via GUI
+- See with one view in Billing/Export in Tabs how many invoices you have to create
```
Comment (kevinpapst) [inline _store/hmr-it-controlling-bundle.md:70]: Invoices for what?
Clarify where Kimai feature ends and your plugin starts

File: _store/hmr-it-hasworked-bundle.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+---
+title: HasWorked Plugin
```
Comment (kevinpapst) [inline _store/hmr-it-hasworked-bundle.md:2]: Plugin lowercase = plugin (to be consistent with all other store items)

File: _store/hmr-it-hasworked-bundle.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+name: HasWorked Plugin (notify users if they haven't entered times)
```
Comment (kevinpapst) [inline _store/hmr-it-hasworked-bundle.md:3]: Plugin lowercase = plugin (to be consistent with all other store items)

File: _store/hmr-it-hasworked-bundle.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+intro: "A Kimai2 plugin to send emails not users that don't have any entries on the current day yet"
```
Comment (kevinpapst) [inline _store/hmr-it-hasworked-bundle.md:4]: that sentence sounds strange (check not & yet)

File: _store/hmr-it-recordbook-bundle.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
+---
+title: RecordBook Plugin
```
Comment (kevinpapst) [inline _store/hmr-it-recordbook-bundle.md:2]: Plugin lowercase = plugin (to be consistent with all other store items)

File: _store/hmr-it-recordbook-bundle.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+name: RecordBook Plugin (Berichtsheft für Azubis)
```
Comment (kevinpapst) [inline _store/hmr-it-recordbook-bundle.md:3]: Plugin lowercase = plugin (to be consistent with all other store items)

File: _store/hmr-it-recordbook-bundle.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+    name: "RecordBook"
+    screenshots:
+     - 
+        src: "/images/marketplace/hmr-it-recordbook-pdf.png"
+        title: "PDF example" 
+        description: "PDF example" 
+     - 
+        src: "/images/marketplace/hmr-it-recordbook-view.png"
+        title: "Download view"
+        description: "Trainees see the weeks and can download the pdf"
```
Comment (kevinpapst) [inline _store/hmr-it-recordbook-bundle.md:23]: pdf = PDF

**Review by kevinpapst — CHANGES_REQUESTED** [review]
Nice additions 👍 lets make the pages a bit more shiny 😄 and then it can be published

sorry for the long waiting time, I forgot about the PR

File: _store/hmr-it-controlling-bundle.md [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+
+## Features
+
+- See if your users have overtimes or undertimes (since first day in company, per month, per day - all in separate views)
+- Check if they violate the German break time law "Arbeitszeitgesetz (ArbZG) § 4 Ruhepausen"
+- Users see how much overtimes or undertimes they have + how much they've worked already in the current month and how much they still have to work in the current month
+- Filter data per user, time period, and so on
+- E-Mail to user to notify him, that he has violated the German break time law
+- Overview for Administrators to see all calculated times in one view
+- Get notified if one of your user's birthday is coming soon
```
[the remark this answers is not in the document]
Comment (kevinpapst) [reply _store/hmr-it-controlling-bundle.md:62]: See Requires section, understand!

**Review by kevinpapst — APPROVED** [review]

**Comment by kevinpapst** [comment]
Muchas gracias for extending the Kimai plugin family!

**Pull request merged**